CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 Refresh Documentation
 Add NGSIv2 metadata support to device provisioned attributes
 Fix: Error message when sending measures with unknown/undefined attribute
-Add Null check within executeWithSecurity() to avoid crash (#829)
\ No newline at end of file
+Add Null check within executeWithSecurity() to avoid crash (#829)
+Add NGSIv2 metadata support to attributeAlias plugin.
```

Comment:
This line seems to belong to PR #839. Thus, I understand that jason-fox:feature/ngsi-ld-measure is a branch of jason-fox:feature/add-v2-metadata.

PR #839 should be merged before, then this PR updated.

Reply:
Correct - this PR is dependent upon  and extends the metadata change.

Reply:
PR #839 has been merged. This PR branch should be upgrade, then this diff will disappear.

Reply:
I've merge master which should reduce the diff - Fixed with dd81751